storefront/common/components/Layout.tsx

```diff
@@ -12,13 +13,25 @@ export default function Layout({ children }: Props) {
       <Head>
         <title>Yas - Storefront</title>
         <meta name="description" content="Yet another shop storefront" />
+        <link
+          rel="stylesheet"
```

Comment:
please move this import to _app.tsx

Reply:
thanks for your suggestion. i had move bootstrap CDN to proper file, please have a check